src/main/java/com/softdevsix/api/services/report/ReportService.java

```diff
@@ -1,15 +1,24 @@
 package com.softdevsix.api.services.report;
 
+import com.softdevsix.api.domain.entities.file.File;
+import com.softdevsix.api.domain.entities.project.Project;
 import com.softdevsix.api.domain.report.Report;
+import com.softdevsix.api.mappers.json.ProjectMapper;
+import com.softdevsix.api.repositories.IFileRepository;
+import com.softdevsix.api.repositories.IProjectRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ReportService implements IReportService {
 
     private final JsonReportReader reportReader;
+    private final IProjectRepository projectRepository;
```

Comment:
I advise you to use `@Value` since `@Value` has the private and the final. This at the beginning of the class.